### ISSUE TITLE
Problem with detecting existing issues

### ISSUE THREAD

**Issue by dmnd** [atom/notifications#25]
See AtomLinter/Linter#343. There's been a few duplicate issues opened even though there is an existing open issue. Probably something to do with searching for existing issues is not working correctly.

**Comment by matiasgarciaisaia** [comment]
I've just reported the same on atom/deprecation-cop#21 and been told to report it here. I didn't know it was implemented at all - it just doesn't seem to work :/

Do you have any idea of why this could be happening? Can we help anyhow?

**Comment by benogle** [comment]
@matiasgarciaisaia this is slightly different. Deprecation cop doesnt search for issues, iirc. Notifications does, but it's not doing it correctly in some cases. I'm going to reopen your issue on dep cop.

**Comment by stramel** [comment]
I have seen the notifications showing a "View Issue" button for notifications that arose yesterday on 0.175.0. However, the deprecation cop view still seems to only show a create issue link on 0.176.0-f7ecc3e. Am I missing something where the notifications plugin is in charge of displaying stuff within the deprecation cop view?

**Comment by benogle** [comment]
Deprecation cop is separate code. It does not search for an existing issue.

**Comment by stramel** [comment]
@benogle so this issue is resolved, correct?

**Comment by benogle** [comment]
No. It has always searched, but this issue suggests the searching is not working in some cases.

**Comment by dirk-thomas** [comment]
I just ran into this problem. For the case where the issue title contains a path under the `.atom` folder I proposed a patch to replace this user specific path with a generic placeholder in #113. Any feedback is welcome.
